jobs/enthought_web-developer.html

```diff
@@ -0,0 +1,41 @@
+---
+title: Full Stack Web Developer 
+company: Enthought
+url: http://www.enthought.com/company/careers/
+location: Cambridge, UK
+contract: permanent
+contact:
+    name: Jill Cowan
+    email: jobs@enthought.com
+    phone: 
```

Comment:
Consider adding a phone number, or alternatively just leave this line out altogether.